Buildings/Experimental/DHC/CentralPlants/Cooling/Examples/Plant.mo

```diff
@@ -160,12 +164,16 @@ Buildings.Experimental.DHC.CentralPlants.Cooling.Plant</a>.
       revisions="<html>
 <ul>
 <li>
+October 20, 2021, by Chengnan Shi:<br/>
+Revise the model for extensibility.
```

Comment:
change the revision note as, `Revised the model for extensibility. This is for <a href=\"https://github.com/lbl-srg/modelica-buildings/issues/2749\">issue #2749</a>.`

Reply:
Thank you Dr. Hu. Will do!